doc/source/Container.md

```diff
@@ -0,0 +1,151 @@
+```
+$ export CLUSTER_NAME=Watson
+$ export CONVERSATION_SERVICE=conversation-service-watson-pizzeria
+$ export CONFIG_MAP=watson-pizzeria-config
+$ export POD_NAME=watson-pizzeria-pod.yml
+$ export KUBE_SERVICE=pizza-bot
+$ export CONTAINER_ENV_VARIABLE=CONVERSATION_SERVICE_WATSON_PIZZERIA
+```
+
+
+# Steps
+
+## Create the Kubernetes cluster
+
+* Follow the instructions to [Create a Kubernetes Cluster](https://github.com/IBM/container-journey-template).
+```
+$ bx cs cluster-create --name $CLUSTER_NAME
+```
+
+## Create the Watson Conversation Service and bind to your cluster
+
+Either follow the instructions to [Create a Conversation Service](https://console.ng.bluemix.net/catalog/services/conversation) or perform the following from the Bluemix CLI.
+
+* Create the Watson Conversation service:
+
+```
+$ bx service create conversation free $CONVERSATION_SERVICE
+```
+
+
+* Verify that the service instance is created:
+
+```
+$ bx service list
+```
+
+* Obtain the ID of your cluster:
+
+```
+$ bx cs clusters
+```
+
+* Bind the service instance to your cluster:
+
+```
+$ bx cs cluster-service-bind <cluster-ID> $CONVERSATION_SERVICE
+```
+
+## Load the Watson Conversation
+
+* Launch the **Watson Conversation** tool. Use the **import** icon button on the right:
+
+<p align="center">
+  <img width="50%" height="50%" src="images/import_conversation_workspace.png">
+</p>
+
+* Find the local version of [`data/pizza-advanced.json`](data/pizza-advanced.json) and select
+**Import**. Find the **Workspace ID** by clicking on the context menu of the new
+workspace and select **View details**:
+
+<p align="center">
+  <img src="images/open_conversation_menu.png">
+</p>
+
+export the workspace_id:
+
+```
+export <workspace_id>=WORKSPACE_ID
+```
+
+## Create a Kubernetes Configuration Map with the Workspace ID
+
+* Create a Kubernetes Configuration Map:
+
+```
+$ kubectl create configmap $CONFIG_MAP \
+    --from-literal=workspace_id=$WORKSPACE_ID
+```
+
+* Verify that the configuration is set:
+
+```
+$ kubectl get configmaps $CONFIG_MAP -o yaml
+```
+
+## Deploy the Pod
+
+* Deploy:
+
+```
+$ kubectl create -f $POD_NAME
+```
+
+* Identify the **Public IP** address of your worker:
+
+```
+$ bx cs workers $CLUSTER_NAME
+``` 
+
+* Identify the external port your pod is listening on: 
+
+>> Note: The Dockerfile determines the port that the container listens on using the `EXPOSE <port>` command. Kubernetes maps this a publicly addressable port:
+
+```
+$ kubectl get services $KUBE_SERVICE
+```
+
+* Access the application using `http://<IP Address>:<Port>`
+
+## Look Under the Hood
+
+Now that you have created and bound a service instance to your cluster, let's take a deeper look at what is happening behind the scenes.
+
+
+* Binding the conversation service to your cluster creates a Kubernetes secret named `binding-${CONVERSATION_SERVICE}` .
+
+* The secret contains a key named binding with its data being a JSON string of the form:
+
+```
```

Comment:
I think this is the one that could benefit (a little) from mentioning the json lang